src/aiokem/main.py

```diff
@@ -38,64 +50,51 @@ def __init__(self, session: ClientSession = None) -> None:
         """Initialize the AioKem class."""
         self._token: str | None = None
         self._refresh_token: str | None = None
-        self._session = (
-            session if session else ClientSession(timeout=ClientTimeout(total=10))
-        )
-        _LOGGER.info("AioKem instance initialized.")
+        self._session = session or ClientSession(timeout=CLIENT_TIMEOUT)
 
-    async def _authentication_helper(
-        self, headers: CIMultiDict, data: CIMultiDict
-    ) -> None:
+    async def _authentication_helper(self, data: dict[str, Any]) -> None:
+        _LOGGER.debug("Sending authentication request to %s", AUTHENTICATION_URL)
         try:
-            _LOGGER.debug("Sending authentication request to %s", AUTHENTICATION_URL)
             response = await self._session.post(
-                AUTHENTICATION_URL, headers=headers, data=data
+                AUTHENTICATION_URL, headers=AUTH_HEADERS, data=data
             )
             response_data = await response.json()
-            if response.status != HTTPStatus.OK:
-                if response.status == HTTPStatus.BAD_REQUEST:
-                    raise AuthenticationCredentialsError(
-                        f"Invalid Credentials: "
-                        f"{response_data.get('error_description', 'unknown')} "
-                        f"Code {response.status}"
-                    )
-                else:
-                    raise AuthenticationError(
-                        f"Authentication failed: "
-                        f"{response_data.get('error_description', 'unknown')} "
-                        f"Code {response.status}"
-                    )
-            self._token = response_data.get("access_token")
-            if not self._token:
-                raise Exception("Login failed: No access token received")
-
-            self._refresh_token = response_data.get("refresh_token")
-            if not self._refresh_token:
-                raise Exception("Login failed: No refresh token received")
-
-            self._token_expiration = response_data.get("expires_in")
-            self._token_expires_at = datetime.now() + timedelta(
-                seconds=self._token_expiration
-            )
-            _LOGGER.info(
-                "Authentication successful. Token expires at %s", self._token_expires_at
-            )
-
         except ClientConnectionError as e:
             raise CommunicationError(f"Connection error: {e}") from e
 
-    async def authenticate(self, username: str, password: str) -> None:
-        """Login to the server."""
-        _LOGGER.info("Authenticating user %s", username)
-        headers = CIMultiDict(
-            {
-                "accept": "application/json",
-                "authorization": f"Basic {CLIENT_KEY}",
-                "content-type": "application/x-www-form-urlencoded",
-            }
```

Comment:
Made the headers a constants and used `aiohttp.hdrs` to avoid constructing it every time